base/Base/Base_Base_implementation.F90

```diff
@@ -192,9 +192,12 @@ module subroutine MAPL_FieldAllocCommit(field, dims, location, typekind, &
              allocate(VAR_1D(UNGRID(1)), STAT=STATUS)
              _VERIFY(STATUS)
              VAR_1D = INIT_VALUE
-             call ESMF_FieldEmptyComplete(FIELD, farrayPtr=VAR_1D,    &
+             call ESMF_FieldEmptyComplete(FIELD, farray=VAR_1D,    &
+                  indexflag=ESMF_INDEX_DELOCAL, &
                   datacopyFlag = ESMF_DATACOPY_REFERENCE,             &
                   gridToFieldMap=gridToFieldMap,                      &
+                  ungriddedLBound=[1],&
+                  ungriddedUBound=[ungrid(1)], &
                   rc = status)
```

Comment:
It looks like `status` here is not being checked.  As long as we are here, please fix this.

Reply:
Satisfied in 224 at the end of the 'if' block